ui/apps/platform/src/services/AdministrationEventsService.ts

```diff
@@ -0,0 +1,83 @@
+import axios from './instance';
+
+import { Pagination } from './types';
+
+const eventsCountUrl = '/v1/count/administration/events';
+const eventsUrl = '/v1/administration/events';
+
+export type AdministrationEvent = {
+    id: string;
+    type: AdministrationEventType;
+    level: AdministrationEventLevel;
+    message: string;
+    hint: string;
+    domain: string;
+    resourceType: string;
+    resourceId: string;
+    numOccurrences: string; // int64
```

Comment:
A numeric value returned as a string? 😢

Reply:
Yeah, maximum safe integer is 2 ** 53 so `int64` in proto corresponds to string in response.

Reply:
Aha, that does make sense on some level. We still lose precision when changing type via `Number()` though right, if the value is over the max safe integer?

Although I suppose if we have over `9007199254740991` data items the loss in precision isn't as dire. 😄

Reply:
We just render `numOccurrences` so number versus string does not matter.

I asked in the feature channel about the count because pagination payload has `int32` type so `int64` seems like as much a problem for backend as frontend. With event aggregation and default 4 days retention, assuming less than 2 ** 31 discrete events seems as reasonable as for violations.